assets/src/edit-story/components/library/panes/media/mediaPane.js

```diff
@@ -252,6 +253,24 @@ function MediaPane(props) {
     [hasMore, isMediaLoading, isMediaLoaded, setNextPage]
   );
 
+  // The media pane is supposed to have a scrollbar that shows on focus and
+  // "hovers" over content. As of May 2020 this cannot be achieved without
+  // js (as the scrollbar-gutter prop is no yet ready).
+  useEffect(() => {
+    if (!refContainer.current) {
+      return;
+    }
+    const scrollBarWidth =
+      refContainer.current.offsetWidth - refContainer.current.clientWidth;
+    const currentPaddingLeft = parseFloat(
+      window
+        .getComputedStyle(refContainer.current, null)
+        .getPropertyValue('padding-left')
+    );
+    refContainer.current.style['padding-right'] =
+      currentPaddingLeft - scrollBarWidth + 'px';
+  });
```

Comment:
This gets called many times, should this have a dependency array?

Maybe something like this: https://medium.com/tech-eatwith/how-to-use-react-refs-with-useeffect-hooks-9a0b31a7b398

Reply:
Done, I used a callback ref and added the necessary dependencies. Ended up being a bit more verbose.